payjoin-cli/contrib/test.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -e
 
-cargo test --locked --package payjoin-cli --verbose --all-features
+cargo test --locked --package payjoin-cli --verbose
```

Comment:
This removes tests run by this file since `v1` isn't included in the `default` feature set.